ping/_compositions/go-rust-interop-latest.toml

```diff
@@ -14,38 +14,26 @@
     go_proxy_url           = "https://proxy.golang.org"
 
 {{ with (load_resource "./go.toml") }}
-  {{ with .master }}
+  {{ with (index .groups 0) }}
```

Comment:
@marten-seemann No worries, I requested your review regarding the feature itself,

nothing much to see in the code, quick overview:
* we use go templates to simplify maintenance,
* with `load_resource` we load https://github.com/libp2p/test-plans/blob/master/ping/_compositions/go.toml in the template.

This PR replaces the use of the `master` version with the latest releases registered in `.groups`,